src/Kdyby/PayPalExpress/PayPal.php

```diff
@@ -251,7 +251,7 @@ private function process($data)
 
 		$request = new Curl\Request($this->host, $data);
 		$request->setSender($this->curlSender);
-		$request->options['sslversion'] = 3;
+		$request->options['sslversion'] = 1; // 1 means CURL_SSLVERSION_TLSv1, but in some PHP versions this constant is not defined (https://bugs.php.net/bug.php?id=62318)
```

Comment:
Probably yes. Official PayPal integration wizard generates PHP code without setting this CURL option. I think if option is not set, it's like using constant CURL_SSLVERSION_DEFAULT. So I will remove this line.